src/components/DiseaseTable.tsx

```diff
@@ -24,6 +24,7 @@ const {
     clones,
     cellLineId,
     expandableContent,
+    hoveredRow,
```

Comment:
I think `hoverColumn` could be renamed to keep the style sheet from being confusing now that we have hoverRow. `actionColumn` seems good?